build.gradle

```diff
@@ -134,8 +130,6 @@ publishing {
             artifactId = 'java-client'
             version = '7.3.0'
             from components.java
-            artifact sourcesJar
```

Comment:
why documentation and sources are excluded?

Reply:
components.java should include both sources and Javadoc as specified [here](https://github.com/appium/java-client/pull/1389/files#diff-c197962302397baf3a4cc36463dce5eaR33).